packages/gitbook/src/components/DocumentView/Table/RecordColumnValue.tsx

```diff
@@ -130,8 +130,9 @@ export async function RecordColumnValue<Tag extends React.ElementType = 'div'>(
                         'lg:space-y-3',
                         'leading-normal',
                         verticalAlignment,
-                        alignment === 'right' ? 'text-right' : null,
-                        alignment === 'center' ? 'text-center' : null,
+                        alignment === 'right' ? '[&_*]:text-right' : null,
+                        alignment === 'center' ? '[&_*]:text-center' : null,
+                        alignment === 'left' ? '[&_*]:text-left' : null,
```

Comment:
```suggestion
                        alignment === 'right' ? '*:text-right' : null,
                        alignment === 'center' ? '*:text-center' : null,
                        alignment === 'left' ? '*:text-left' : null,
```

I think it's equivalent, also as I said to @clairechabas it would be better to use an extensive switch to statically check that if we add an alignment we don't forget that place in the code.

Reply:
Like this: https://github.com/GitbookIO/gitbook/pull/3175/commits/bb8b4aa4487eeb168eca7e4fa33149e2ae8533cb?

Also, I checked but *: does not impact the child node while `[&_*]` seems to be better supported by tailwind.